### PR TITLE
[ITT-208] Clean up code

### PR DESCRIPTION
**Pull request by antolo-arch**
**Description**
Remove the '/' replacement in the template, regenerate all services, fix unit tests

### CODE REVIEW

File: Adyen/client.py [hunk 1]
```diff
@@ -379,14 +379,14 @@ def call_adyen_api(
...
-            "authorise3ds2",
-            "payments",
-            "paymentSession",
-            "paymentLinks",
-            "paymentMethods/balance",
-            "sessions"
+            "/authorise",
+            "/authorise3d",
+            "/authorise3ds2",
+            "/payments",
```
Comment (jillingk) [inline Adyen/client.py:385]: 👍

File: Adyen/services/binLookup.py [hunk 2]
```diff
@@ -22,12 +12,11 @@ def __init__(self, client=None):
         super(AdyenBinlookupApi, self).__init__(client=client)
         self.service = "binlookup"
 
-    def get_3ds_availability(self, request, idempotency_key=None, **kwargs):
+    def get3ds_availability(self, request, idempotency_key=None, **kwargs):
```
Comment (jillingk) [inline Adyen/services/binLookup.py:15]: Was this a spec change or manual?

File: Adyen/services/checkout/utility_api.py [hunk 3]
```diff
@@ -17,16 +17,14 @@ def get_apple_pay_session(self, request, idempotency_key=None, **kwargs):
         Get an Apple Pay session
         """
         endpoint = f"/applePay/sessions"
-        endpoint = endpoint.replace('/', '', 1)
         method = "POST"
         return self.client.call_adyen_api(request, self.service, method, endpoint, idempotency_key, **kwargs)
 
-    def create_originkey_values_for_domains(self, request, idempotency_key=None, **kwargs):
+    def origin_keys(self, request, idempotency_key=None, **kwargs):
```
Comment (jillingk) [inline Adyen/services/checkout/utility_api.py:23]: 👍

**Review by jillingk — APPROVED** [review]
gj dude

File: Adyen/services/binLookup.py (hunk 2 above)
Comment (antolo-arch) [reply Adyen/services/binLookup.py:15]: spec

**Review by AlexandrosMor — APPROVED** [review]

**Pull request merged**